data/json/npcs/talk_tags.json

```
@@ -777,7 +909,12 @@
       "Hey, <name_g>!  I've got",
       "<name_g>!  Watch my back while I kill",
       "I'm your huckleberry,",
-      "Sorry, but you have to do down,"
+      "Sorry, but you have to do down,",
```

```suggestion
      "Sorry, but you have to go down,",
```
Not your change, but there's a typo here.